lib/win32/certstore/store_base.rb

```diff
@@ -135,6 +135,20 @@ def read_certificate_content(cert_path)
         end
         File.read("#{cert_path}")
       end
+
+      def find_certificate(store_handler, certificate_name)
+        issuer_rdn_name = FFI::MemoryPointer.new(2, 256)
+        while (pCertContext = CertEnumCertificatesInStore(store_handler, pCertContext) and not pCertContext.null?)do
+          CertGetNameStringW(pCertContext, CERT_NAME_RDN_TYPE, CERT_NAME_ISSUER_FLAG, nil, issuer_rdn_name, 5000)
+          rdn_name_from_store = issuer_rdn_name.read_wstring.downcase.gsub(/, /, ',').split(',')
+          rdn_name_from_user = certificate_name.downcase.gsub(/, /, ',').split(',')
+          if( (rdn_name_from_store - rdn_name_from_user).empty? and (rdn_name_from_user - rdn_name_from_store).empty? )
+            return pCertContext
+          end
+        end
+        lookup_error if(pCertContext.null?)
+      end
+
```

Comment:
can you fix the whitespacing here to ruby style?

https://github.com/bbatsov/ruby-style-guide#spaces-braces

`do` on line 141 should have a space before it. Your `if` blocks don't have a whitespace after the `if` but your `while` does. You shouldn't need parenthesis on your if blocks at all.

You should prefer `&&` over `and` https://github.com/bbatsov/ruby-style-guide#no-and-or-or.